lib/matplotlib/backends/__init__.py

```diff
@@ -7,29 +7,49 @@
 import inspect
 import warnings
 
-# ipython relies on interactive_bk being defined here
-from matplotlib.rcsetup import interactive_bk
 
-__all__ = ['backend','show','draw_if_interactive',
-           'new_figure_manager', 'backend_version']
```

Comment:
Most of these things are no longer exist in the module so it would raise an AttributeError.